docs/content/operator/common/monitoring.mdx

```diff
@@ -0,0 +1,102 @@
+After a successful setup of a node, the next important step is to set up monitoring for the node.
+Monitoring is important to keep track of the node's health and performance.
+
+## Network Monitoring
+All active validators for the testnet are listed on the [IOTA Explorer](https://explorer.rebased.iota.org/validators) website.
+
+### Health Check Endpoints
+Some of the important checks are below:
+
+```sh
+curl -s localhost:9184/metrics | grep -E "^last_executed_checkpoint|^highest_synced_checkpoint|^highest_known_checkpoint|^last_committed_round|^consensus_threshold_clock_round|^highest_received_round|^consensus_proposed_blocks|^uptime"
+```
+
+```sh
+highest_known_checkpoint 43448601
+highest_synced_checkpoint 43448600
+last_executed_checkpoint 43448601
+last_executed_checkpoint_age_ms{pct="50"} 188
+last_executed_checkpoint_age_ms{pct="95"} 221
+last_executed_checkpoint_age_ms{pct="99"} 297
+last_executed_checkpoint_age_ms_count 287523
+last_executed_checkpoint_age_ms_sum 4291692609431
+last_executed_checkpoint_timestamp_ms 1741700144842
+uptime{chain_identifier="2304aa97",process="fullnode",version="0.10.3-rc-c2adf3fd68e9"} 20925
+```
+
+```sh
+curl --json '{"jsonrpc":"2.0","id":1,"method":"iota_getLatestCheckpointSequenceNumber","params":[]}' localhost -s | jq .result
+```
+
+```sh
+"43428454"
+```
+
+### Node Metrics
+
+IOTA nodes expose a wide range of metrics to be scraped by [Prometheus](https://prometheus.io/).
+By default, metrics are available at the `http://localhost:9184/metrics` endpoint.
+The best way to visualize these metrics is to use [Grafana](https://grafana.com/).
+Additionally, a common approach is to use [node exporter](https://github.com/prometheus/node_exporter) to scrape performance metrics from the node and push them to Prometheus.
+
+### Monitoring Services
+
+Implementing monitoring services is essential to ensure the reliability, security, and performance of the blockchain network by providing real-time insights, detecting anomalies, enabling proactive issue resolution, and receiving automatic alerts.
+
+**Prometheus and Grafana**  
+Currently, we do not have an official Grafana setup guide.
```

Comment:
Sounds like a promise that this would change in the future.
Just embrace the work of the community directly.

Reply:
Removed this sentence in 0c0a6337ba4b90bca65d1713222fc05d4843e3bf